dali/pipeline/executor/executor_test.cc

```diff
@@ -600,11 +590,16 @@ TYPED_TEST(ExecutorSyncTest, TestPrefetchedExecution) {
   ASSERT_EQ(ws.NumInput(), 0);
   ASSERT_TRUE(ws.OutputIsType<GPUBackend>(0));
   TensorList<GPUBackend> &res1 = ws.Output<GPUBackend>(0);
-  for (int i = 0; i < batch_size; ++i) {
-    this->VerifyDecode(
-        res1.template tensor<uint8>(i),
-        res1.tensor_shape(i)[0],
-        res1.tensor_shape(i)[1], i);
+
+  TensorList<CPUBackend> res_cpu;
+  res_cpu.Copy(res1, cudaStream_t(0));
```

Comment:
```suggestion
  res_cpu.Copy(res1);
```
I think you can use just the default ordering.